common/k8s/src/middleware/metadata.rs

```diff
@@ -9,12 +9,17 @@ use kube::{api::ListParams, config::Config, Api, Client};
 use kube_runtime::watcher;
 use kube_runtime::watcher::Event as WatcherEvent;
 
+use backoff::backoff::Backoff;
+use backoff::ExponentialBackoff;
 use metrics::Metrics;
 use middleware::{Middleware, Status};
 use parking_lot::Mutex;
+use pin_utils::core_reexport::option::Option::Some;
```

Comment:
Do we need to pull this in? It appears to just be core::option::Option, which is already imported as part of the rust prelude (via it's other re-export in std::option::Option)

Reply:
Good catch. It was probably auto imported by the IDE... I'll remove it.